tee-worker/local-setup/worker-config-local.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
why add one more mode here? Is 'dev/staging/prod' not enough ?

Reply:
`dev` equals `local`, so this config file should not be added.

Reply:
I misunderstand, I see tdf in the dev  configuration, so add one. I will delete it later

Reply:
Hmmmm, I think the `dev` configuration is different from the `local` configuration, and the `local` configuration is more used in ts-test. We can rename `local` to `mock`, which looks more intuitive

Reply:
Hmmm, also make sense. Let's rename it to `mock`.

Reply:
Done, pls check.